### DISCUSSION TITLE
Is Partial Acceptance of Suggestions Using Ctrl + RightArrow Key Possible in Visual Studio 2022?

### DISCUSSION THREAD

**Discussion by PlusoneWang** [community/community#103070] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I've noticed that in Visual Studio Code, I can use the `Ctrl` + `RightArrow` Key combination to partially accept suggestions, which is quite handy. However, this functionality doesn't seem to be available in Visual Studio 2022. Often, I only need the first few words from a suggestion. Currently, I'm forced to accept the entire suggestion provided by Copilot, then delete the unnecessary words or lines. This process can be cumbersome, and there are times when manually typing it out seems faster. Is there a way to partially accept suggestions in Visual Studio 2022 similar to how it's done in Visual Studio Code?

**Comment by ayim** [comment, marked as answer]
Nothing I can announce at the moment in this product area but it's something we're thinking about and experimenting with.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
github: ayim | aayim@microsoft.com

**Comment by AlexFrancoMx** [comment]
Any update about this?

**Comment by looak** [comment]
I agree with Alex, bump! I saw there was a new version released today but it doesn't mention anything about partial accepting suggestions. A feature I really enjoy in VSCode.

**Comment by ThomasNewtonII** [comment]
I really like having this feature in VSCode and it would be great to have it in VS Studio as well. It does feel a bit odd that the free to use offering has this feature but the paid enterprise IDE does not.  That being said I understand there could be a myriad of reasons why this is the case.

**Comment by namnt520** [comment]
Hopefully this feature can be implemented soon. I think it's kind of helpful in real cases.
